server/server.go

```diff
@@ -1993,10 +1993,15 @@ func (s *Server) checkPathScope(ctx context.Context, txn storage.Transaction, pa
 	}
 
 	spath := strings.Trim(path.String(), "/")
+	spathParts := strings.Split(spath, "/")
+
+	if spath == "" && len(bundleRoots) > 0 {
```

Comment:
Nit: This check can be moved before the `Split`.

Reply:
Fixed.